app/models/project.rb

```diff
@@ -1,4 +1,15 @@
 class Project < ApplicationRecord
   has_one :song
-  
+
+  after_save :init_metadata
+
+
+  # todo add more default fields here
+  def init_metadata
```

Comment:
these functions bind the mongo model to this AR model. I found some gems that would do this fancier, but they weren't maintained anymore so fuck that. in any case since it's 1-1 association this is fine i bet.